tailwind/plugins/base/gc-base.js

```diff
@@ -7,6 +7,7 @@ module.exports = function () {
     addBase({
       html: {
         overflowY: 'visible !important', // Need this for sticky nav to work
+        color: '#17171A', // Campaign black
```

Comment:
Decanter default text color is Decanter black. Update this to Campaign black for this project as the fall back text color